src/main/java/uk/gov/companieshouse/model/Reason.java

```diff
@@ -7,6 +7,7 @@
 
     @Valid
     private OtherReason other;
+    @Valid
```

Comment:
IllnessReason currently has no validation criteria. Could you add them?
https://github.com/companieshouse/lfp-appeals-api/blob/master/src/main/java/uk/gov/companieshouse/model/IllnessReason.java

Reply:
Ops, done!